airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -802,7 +802,8 @@ def _get_records(self, pull_request, repository_name):
             record["pull_request_url"] = pull_request["url"]
             if record["commit"]:
                 record["commit_id"] = record.pop("commit")["oid"]
-            record["user"]["type"] = record["user"].pop("__typename")
+            if record["user"] is not None:
```

Comment:
```suggestion
            if record["user"]:
```
It achieves the same result and if the `record["user"]` doesn't exist it will break the code. A more defensive code is:
```python
if "user" in record.keys():
    # do stuff
```

Reply:
Good catch! The downside of checking `record.keys()` is we'll have an error if `record['user'] == undefined`. Let's use your first suggestion.